circle.yml

```diff
@@ -0,0 +1,31 @@
+machine:
+  node:
+    version: v4.2
+  ruby:
+    version: 2.2.3
+dependencies:
+  pre:
+    # Setup for CodePush
+    - npm install -g cordova code-push-cli
+    - code-push login --accessKey $CODE_PUSH_ACCESS_KEY
+    # Setup cordova for generating build
+    - ionic state restore
```

Comment:
Any idea why this command would fail if we move it to `deployment` section? We had that issue with sprout. Ideally that's where we would have this run.